lang_portal_backend/lang_portal_backend/docker_settings.py

```
@@ -0,0 +1,21 @@
+"""
+Django settings for lang_portal_backend project - Docker configuration.
+"""
+
+from .settings import *
```

Ruff (0.8.2)

`from .settings import *` used; unable to detect undefined names

(F403)